scripts/list-images.py

```diff
@@ -14,12 +14,21 @@
     parser = argparse.ArgumentParser(
         description='Kogito Version Manager - List Images by Community and Product version')
     parser.add_argument('--prod', default=False, action='store_true', help='List product images')
+    parser.add_argument('-s', '--supporting-services', default=False, action='store_true',
+                        help='List Supporting Services images')
+    parser.add_argument('-is', '--is-supporting-services', default=False, type=str,
+                        help='Query the given image, if found exit 0, otherwise exit 10.')
 
     args = parser.parse_args()
 
     images = []
     if args.prod:
         images = common.get_prod_images()
+    elif args.supporting_services:
+        if args.is_supporting_services:
+            exit(common.is_supporting_services_image(args.is_supporting_services))
+        else:
+            images = common.get_supporting_services_images()
```

Comment:
I was meaning having this behavior:

```suggestion
    elif args.supporting_services:
        images = common.get_supporting_services_images()
    elif args.is_supporting_services:
        exit(common.is_supporting_services_image(args.is_supporting_services))    
```